README.md

```diff
@@ -1,26 +1,101 @@
+- Verify the Heroku command line tool has been installed in the development environment. [Heroku CLI Documentation](https://devcenter.heroku.com/articles/heroku-cli#install-with-ubuntu-debian-apt-get)
+- PostgreSQL: The default Rails development environment expects to use the SQLite3 database. For shifting to PostgreSQL for development, follow the instructions in this [guide](https://github.com/sidbav/public-health-app-v2/wiki/Shifting-to-PostgreSQL-for-development).
+
+
+## Installing
+
+Clone the repository:
+`git clone https://github.com/sidbav/public-health-app-v2.git`
+
+Install the dependencies:
+bundle install
+
+Verify you have Ruby, Rails, and Heroku CLI installed:
+ruby -v &&\
+rails -v &&\
+bundle -v &&\
+heroku -v &&\
+
+### Setup the database:
+`rails db:create`
+`rails db:migrate`
+
+
+### To seed the database, run:
+`rails db:seed`
+
+### Running the Application
+You can start the rails server using the command given below.
+`rails s`
+
+### Change the database for production
+- All apps on Heroku use the PostgreSQL database. For Ruby/Rails apps to do so, they must include the pg gem.To specify production-specific gems, add `gem 'pg'`  for Heroku deployment
+- To make sure sqlite3 gem is only loaded in development and test environments but not production.
+ make sure  `gem 'sqlite3'` is added for development
+
+- Always Run bundle install to update dependencies after you modify gemfile
+
+If you encounter an error, run `bundle config set --local without 'production'` and then `bundle install` again.Commit the updated Gemfile and Gemfile.lock.
+
+Note: Heroku ignores config/database.yml and enforces the use of PostgreSQL. Modifying the production: section in database.yml will not affect Heroku's behavior.
+
+You may find `PostgreSQL Explorer` VS code extension useful 
 
 ## How to run the test suite
 
-## Services (job queues, cache servers, search engines, etc.)
+ 1. Before running the tests, make sure you have the required gems installed. Run `bundle install`.
+ 2. Cucumber is used for testing the application using BDD.  We have defined 31 scenarios to simulate user interactions with the app.To run Cucumber tests `bundle exec cucumber`
+ 3. RSpec is used for TDD in our application.We have 18 RSpec tests covering various aspects of the app. Run `bundle exec rspec`
+ ### Code Coverage with SimpleCov
+ SimpleCov is used for code coverage. To generate a coverage report, add SimpleCov to your test helper file:
+  `require 'simplecov'`
+  `SimpleCov.start`
+ After running tests, view the coverage report by opening coverage/index.html in your browser.
+
+## Heroku Deployment Instructions
+
+1. Authenticate with Heroku CLI:
+   - Run `heroku login` and follow the prompts.
+   - For terminal without a browser, follow [these steps](https://devcenter.heroku.com/articles/authentication#logging-in-without-a-browser).
+
+2. Navigate to your app's root directory.
+
+3. Create a new app "container" on Heroku using `heroku create healthapp`.
```

Comment:
dont we already have heroku deployment instructions in the wiki, why are we also including them here? Probably best to not repeat information. So either keep it here or keep it there.